### ISSUE TITLE
Forum Module

### ISSUE THREAD

**Issue by iyilm4z** [abpframework/abp#1501]
Hello all,

I would like to contribute abp by developing a forum module which includes forums, posts, comments, pm, q&a, etc. However before starting or quitting it, i would like to ask you guys if it's a good idea or not.

Thanks.

**Comment by ismcagdas** [comment]
Do you mean something like https://support.aspnetzero.com ? If so, It is an ABP module which is very similar to your suggestion.

**Comment by iyilm4z** [comment]
Yes definitely something like that. I didn't know abp already has one. Which abp module is it?

**Comment by ismcagdas** [comment]
It is currently private

**Comment by iyilm4z** [comment]
Uhm i see. So you mean, no need the module and i could close this issue?

**Comment by Xeevis** [comment]
Interesting, will it be made public?

**Comment by hikalkan** [comment]
We will have some commercial modules & services on top of the framework. This module will probably be a commercial module.
